pushers/backends/slack/slack.go

```diff
@@ -36,15 +36,14 @@ type Config struct {
 type SlackBackend struct {
 	config Config
 
-	ch chan event.Event
+	ch chan event.Map
```

Comment:
Because Event is the object embeddibg syncmap. While Map is the type for a plain map[string]interface{}.